### ISSUE TITLE
[Feature Request]: New file "Inbox"

### ISSUE THREAD

**Issue by archer31** [TagStudioDev/TagStudio#389]
### Checklist

- [X] I am using an up-to-date version.
- [X] I have read the [documentation](https://github.com/TagStudioDev/TagStudio/blob/main/doc/index.md).
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### Description

Some other file organization softwares include a file "inbox", a place where new files go so that they can be tagged before joining the rest of the files in the store.

### Solution

This could be done by having a default tag of "Inbox" which could be treated like the favorite and archive tags except that new files automatically have this tag applied.

**Comment by CyanVoxel** [comment]
I've been considering this, I think it would end up being pretty beneficial to the tagging workflow. Currently you can search for "untagged" and "empty" entries, but having that extra buffer of an inbox would help in situations where you're not "finished" adding tags or info to an entry before it would no longer turn up in those special searches.

Related to this (and necessary for an inbox) would be the addition of keeping track of when a file was added to your library. "Date Added" could be displayed alongside "Date Created" and "Date Modified" in the preview pane as well.

**Comment by Cool-Game-Gev** [comment]
I think that this is a great idea, but how would the user remove it from the inbox? 

My first thought would be a icon on the preview, **not the thumbnail**, that the user could click to remove the `New`/`Inbox` tag.

**Comment by Nginearing** [comment]
> I think that this is a great idea, but how would the user remove it from the inbox?
> 
> My first thought would be a icon on the preview, **not the thumbnail**, that the user could click to remove a `New`/`Inbox` tag.

Just stumbled upon this now, and I absolutely agree, it could work similarly to how Archived and Favourite are just tags.

**Comment by Cool-Game-Dev** [comment]
> Just stumbled upon this now, and I absolutely agree, it could work similarly to how Archived and Favourite are just tags.
Yes, but I think the icon should disappear after the tag is removed, as its very rare you would put something back in the inbox
